config/locales/en.yml

```diff
@@ -1133,9 +1133,13 @@ en:
       this_is_private_community: "You need to sign up before you can view the content."
     invitation_form:
       email: "Email address(es)"
+      email_with_limit: "Email addresses (max. %{invitation_limit})"
       message: "A personal message"
       send_invitation: "Send invitation"
       add_as_many_emails_as_you_want: "Add the email addresses of the people you are inviting to the field below. If you add multiple email addresses, separate them with comma."
+      add_email_addresses_description: "Add the email addresses of the people you are inviting to the field below. To add multiple email addresses, separate them with comma."
+      add_lots_of_email_addresses: "If you plan to send lots of invitations, you should probably use an emailing tools. Check out %{this_article_link} to find out resources about suitable email services."
```

Comment:
- Should we remove the "maybe". I think the case is pretty clear that if they are sending a lot of email, then the emailing tool should be used.
- "...an emailing tool_", there should not be "s"